core/domain/beam_job_services_test.py

```diff
@@ -82,8 +97,9 @@ def assert_domains_equal_models(self, beam_job_runs, beam_job_run_models):
         """
         self.assertEqual(len(beam_job_runs), len(beam_job_run_models))
         runs = sorted(beam_job_runs, key=lambda j: j.job_id)
-        models = sorted(beam_job_run_models, key=lambda m: m.id)
-        for i, (r, m) in enumerate(python_utils.ZIP(runs, models)):
+        by_id = lambda m: m.id
+        run_models = sorted(beam_job_run_models, key=by_id)
```

Comment:
Can you add a similar comment stating the reason for separating this into 2 lines? Thanks!

Reply:
Done.